enatega-multivendor-app/src/components/Main/RestaurantCard/NewRestaurantCard.js

```diff
@@ -57,6 +57,9 @@ function NewRestaurantCard(props) {
   const handleAddToFavorites = () => {
     if (!loadingMutation && profile) {
       mutate({ variables: { id: props._id } })
+    } else if (!profile) {
+      FlashMessage({ message: t('loginRequired') }) 
+      navigation.navigate('Profile') 
```

Comment:
Isn't there should be CreateAccount instead of Profile ?

Reply:
We are navigating the user to the profile as the profile exists in tab navigations.